src/LowMachEquationSystem.C

```diff
@@ -1569,6 +1598,9 @@ MomentumEquationSystem::register_inflow_bc(
     else if ( fcnName == "kovasznay") {
       theAuxFunc = new KovasznayVelocityAuxFunction(0,nDim);
     }
+    else if ( fcnName == "OneTwoTenVelocity" ) {
+      theAuxFunc = new OneTwoTenVelocityAuxFunction(0,nDim);
```

Comment:
Can you please add a doxygen-style doc in `OneTwoTenVelocityAuxFunction.h` explaining what it is?